managed/ui/src/components/config/Storage/StorageConfigurationContainer.js

```diff
@@ -31,6 +32,21 @@ const mapDispatchToProps = (dispatch) => {
   return {
     addCustomerConfig: (config) => {
       return dispatch(addCustomerConfig(config)).then((response) => {
+        if (response.error) {
+          const errorMessageObject = response.payload?.response?.data?.error || response.payload.message;
```

Comment:
`errroMessage` is fine instead of `errorMessageObject`.

Reply:
Since the actual message is inside it that's why